manifests/init.pp

```diff
@@ -101,53 +101,35 @@
 #     include dns
 #
 class dns (
-  $namedconf_path        = $::dns::params::namedconf_path,
-  $dnsdir                = $::dns::params::dnsdir,
-  $dns_server_package    = $::dns::params::dns_server_package,
-  $rndckeypath           = $::dns::params::rndckeypath,
-  $optionspath           = $::dns::params::optionspath,
-  $publicviewpath        = $::dns::params::publicviewpath,
-  $vardir                = $::dns::params::vardir,
-  $namedservicename      = $::dns::params::namedservicename,
-  $zonefilepath          = $::dns::params::zonefilepath,
-  $localzonepath         = $::dns::params::localzonepath,
-  $forward               = $::dns::params::forward,
-  $forwarders            = $::dns::params::forwarders,
-  $listen_on_v6          = $::dns::params::listen_on_v6,
-  $recursion             = $::dns::params::recursion,
-  $allow_recursion       = $::dns::params::allow_recursion,
-  $allow_query           = $::dns::params::allow_query,
-  $empty_zones_enable    = $::dns::params::empty_zones_enable,
-  $dns_notify            = $::dns::params::dns_notify,
-  $dnssec_enable         = $::dns::params::dnssec_enable,
-  $dnssec_validation     = $::dns::params::dnssec_validation,
-  $namedconf_template    = $::dns::params::namedconf_template,
-  $acls                  = $::dns::params::acls,
-  $optionsconf_template  = $::dns::params::optionsconf_template,
-  $controls              = $::dns::params::controls,
-  $service_ensure        = $::dns::params::service_ensure,
-  $service_enable        = $::dns::params::service_enable,
-  $additional_options    = $::dns::params::additional_options,
-  $additional_directives = $::dns::params::additional_directives,
+  $namedconf_path                                                 = $::dns::params::namedconf_path,
```

Comment:
In the documentation we have a type for every parameter. Could you add those as well?